architecture/additional_concepts/authentication.adoc

```diff
@@ -75,7 +75,9 @@ methods:
 
 OAuth Access Tokens::
 - Obtained from the {product-title} OAuth server using the `_<master>_/oauth/authorize` and `_<master>_/oauth/token` endpoints.
-- Sent as an `Authorization: Bearer...` header or an `access_token=...` query parameter
+- Sent as an `Authorization: Bearer...` header
+- Sent as an `access_token=...` query parameter for websocket requests prior to 3.6
```

Comment:
ifdef::openshift-enterprise[]
prior to {product-title} 3.6.
endif::[]
ifdef::openshift-origin[]
prior to {product-title}1.6.
endif::[]